nav/nav-find-memb-in-mmis.vbs

```diff
@@ -0,0 +1,400 @@
+
+			EMReadScreen relg_elig, 2, relg_row, 33
+			EMReadScreen relg_case_num, 8, relg_row, 73 'reading the case number for this span
+			EMReadScreen relg_status, 1, relg_row+1, 62
+			EMReadScreen relg_start_dt, 8, relg_row+1, 15     'this is where the end date is
+			EMReadScreen relg_end_dt, 8, relg_row+1, 36     'this is where the end date is
+
+			record_elig = False
+			If relg_status = "A" Then record_elig = True
+			If relg_end_dt = "99/99/99" Then
+				record_elig = True
+			ElseIf DateDiff("d", four_months_ago, relg_end_dt) >= 0 Then
+				record_elig = True
+			End If
+
+			If record_elig = True Then
+				ReDim preserve MMIS_HC_SPANS_ARRAY(MMIS_last_const, span_count)
+				MMIS_HC_SPANS_ARRAY(MMIS_pmi_const, span_count) = MMIS_PMI
+				MMIS_HC_SPANS_ARRAY(MMIS_ssn_const, span_count) = rcip_ssn
+				MMIS_HC_SPANS_ARRAY(MMIS_name_const, span_count) = rcip_name
+				MMIS_HC_SPANS_ARRAY(DUPLICATE_pmi_const, span_count) = rcip_duplicate_pmi
+				MMIS_HC_SPANS_ARRAY(MMIS_case_numb_const, span_count) = relg_case_num
+				MMIS_HC_SPANS_ARRAY(MMIS_hc_prog_const, span_count) = relg_prog
+				MMIS_HC_SPANS_ARRAY(MMIS_hc_elig_type_const, span_count) = relg_elig
+				MMIS_HC_SPANS_ARRAY(MMIS_span_start_date_const, span_count) = relg_start_dt
+				MMIS_HC_SPANS_ARRAY(MMIS_span_status_const, span_count) = relg_status
+				MMIS_HC_SPANS_ARRAY(MMIS_span_end_date_const, span_count) = relg_end_dt
+				MMIS_HC_SPANS_ARRAY(string_id_const, span_count) = MMIS_PMI & "-" & relg_case_num & "-" & relg_ssn & "-" & relg_prog & "-" & relg_elig' & "-" &  & "-" &  & "-" &
+				span_count = span_count + 1
+			End If
+
+			relg_row = relg_row + 4         'next span on RELG'
+			If relg_row = 22 Then           'this is the end of RELG and we need to go to a new page
+				PF8
+				relg_row = 6
+				EMReadScreen end_of_list, 7, 24, 26     'This is the end of the list
+				If end_of_list = "NO MORE" Then Exit Do
+			End If
+		Loop
+	Next
+End Function
+Call navigate_to_MAXIS(MX_environment)                          'going back to MAXIS
+
+dlg_len = 30 + 50*(UBound(MMIS_HC_SPANS_ARRAY,2)+1)
+Dialog1 = ""
+BeginDialog Dialog1, 0, 0, 341, dlg_len, "NAV - Find MEMB in MMIS Case Number Dialog"
+	Text 10, 10, 260, 10, "Member: MEMB " & MEMB_ref_numb & " - " & MEMB_last_name & ", " & MEMB_first_name & " - MAXIS PMI: " & MEMB_pmi
+	ButtonGroup ButtonPressed
+		OkButton 285, 5, 50, 15
+
+	y_pos = 25
+	For each_hc_span = 0 to UBound(MMIS_HC_SPANS_ARRAY, 2)
+		GroupBox 10, y_pos, 325, 45, MMIS_HC_SPANS_ARRAY(MMIS_hc_prog_const, each_hc_span) & " - " & MMIS_HC_SPANS_ARRAY(MMIS_hc_elig_type_const, each_hc_span) & " on Case Number: " & MMIS_HC_SPANS_ARRAY(MMIS_case_numb_const, each_hc_span)
+		If MMIS_HC_SPANS_ARRAY(DUPLICATE_pmi_const, each_hc_span) <> "" Then Text 200, y_pos, 150, 10, "DUPLICATE PMI: " & MMIS_HC_SPANS_ARRAY(DUPLICATE_pmi_const, each_hc_span)
+		y_pos = y_pos + 15
+		Text 25, y_pos, 85, 10, "Start Date: " & MMIS_HC_SPANS_ARRAY(MMIS_span_start_date_const, each_hc_span)
```

Comment:
![image](https://github.com/user-attachments/assets/dd9e2870-309c-404c-8551-7f2812ddbbbc)
 
Maybe a separate popup if nothing is found within 4 month window instead of the normal dialog w/ blank entries